components/dashboard/src/projects/new-project/NewProjectRepoList.tsx

```diff
@@ -45,12 +45,9 @@ export const NewProjectRepoList: FC<Props> = ({ filteredRepos, noReposAvailable,
                                         </Button>
                                     ) : (
                                         <p className="text-gray-500 font-medium">
-                                            <a rel="noopener" className="gp-link" href={userLink(r)}>
-                                                @{r.inUse.userName}
-                                            </a>{" "}
-                                            already
+                                            Project already
                                             <br />
-                                            added this repo
+                                            exists.
                                         </p>
```

Comment:
**thought**: Would be nice if we allowed multiple instance of projects per instance, so that users could import projects multiple times. Then, we would be able to link to the project itself. Hoping Projects v2 can helps us with such cases. **No action required.**

![import](https://github.com/gitpod-io/gitpod/assets/120486/cccf5fd7-125f-4f5d-9d10-1bba9607f65f)